tests/examples/awful/popup/wiboxtypes.lua

```diff
@@ -1,6 +1,7 @@
 --DOC_GEN_IMAGE
 --DOC_HIDE_ALL
 --DOC_NO_USAGE
+require("_date")
```

Comment:
`--DOC_HIDE` is not required in this case?

Reply:
Hm, because of `DOC_HIDE_ALL` above?

Reply:
Yup.

The test examples are primarily meant to generate images for the docs. For this, `@DOC_awful_popup_wiboxtypes_EXAMPLE@` is replaced with something. By default, that's all the code used to generate the example and then the example image (or text, in some cases) itself. With `--DOC_HIDE` we can "hide some hacks" and `--DOC_HIDE_ALL` is used for cases where we only want the image (e.g. some overview of all the widgets).